### PR TITLE
prints events from the operator ns on failure

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
please review https://github.com/p0lyn0mial/cluster-kube-apiserver-operator/pull/2 (verifies the encryption cfg) first

File: test/library/encryption/helpers.go [hunk 1]
```diff
@@ -47,8 +47,19 @@ type EncryptionKeyMeta struct {
 	Mode     string
 }
 
+func SetUp(t *testing.T, verboseTearDown ...bool) testing.TB {
```
Comment (sttts) [inline test/library/encryption/helpers.go]: am not a fan of these signatures.  Compare https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis. "variadic configuration" is more future proof.

File: test/library/encryption/helpers.go [hunk 2]
```diff
@@ -282,6 +295,30 @@ func SecretOfLife(t testing.TB) *corev1.Secret {
 	}
 }
 
+func setUpTearDown(verboseOutput bool) func(testing.TB, bool) {
+	return func(t testing.TB, failed bool) {
+		if failed && verboseOutput { // we don't use t.Failed() because we handle termination differently when running on a local machine
```
Comment (sttts) [inline test/library/encryption/helpers.go]: return early if these are not both true

**Comment by p0lyn0mial** [comment]
closing in favour of https://github.com/openshift/library-go/pull/582

**Pull request closed without merging**